src/modules/db_helper.py

```diff
@@ -1,163 +1,225 @@
 import psycopg2
 import psycopg2.extras
+from psycopg2 import sql
 
 SUCCESS = 0
 FAIL = None
 
+def get_table(debug):
+    if debug:
+        return "test"
+    else:
+        return "members"
+
+def connection_error(e, conn):
+    if e is ConnectionError:
+            DATABASE_URL = os.environ['DATABASE_URL']
+            conn = psycopg2.connect(DATABASE_URL, sslmode='require')
+            return FAIL
+    else:
+        conn.rollback()
+        return FAIL
+
+def dbfunc_run(func,tries = 2):

```

Comment:
Let's write this as a function that targets a smaller unit of code. Because the error for database functions is completely within the SQL execution, lets have this function defined as

```
def dbfunc_run(sql, tries=2):
    #other code
    try:
        cur.execute(sql)
    except:
        exception handling
    #other code
    return cur
```

this way we won't have to create a new function every time we want to execute a database function which can get confusing in code having nested functions for every single database function